src/server/pkg/cmdutil/cobra.go

```diff
@@ -48,14 +49,19 @@ func Run(run func(args []string) error) func(*cobra.Command, []string) {
 	}
 }
 
-// ErrorAndExit errors with the given format and args, and then exits.
-func ErrorAndExit(format string, args ...interface{}) {
-	if errString := strings.TrimSpace(fmt.Sprintf(format, args...)); errString != "" {
-		fmt.Fprintf(os.Stderr, "%s\n", errString)
+// This layer of indirection is so we can capture calls to ErrorAndExit in tests
+var internalErrorAndExit = func(message string) {
+	if message != "" {
+		fmt.Fprintf(os.Stderr, "%s\n", message)
 	}
 	os.Exit(1)
 }
```

Comment:
Is there a reason this is a `var` rather than declaring it as just `func internalErrorAndExit(message string) ...` that strikes me as a little more standard.

Also I think it could just be called `errorAndExit` the fact that the first letter is as good as the word internal.